src/table.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as MRE from '@microsoft/mixed-reality-extension-sdk';
+
+export default class Table {
+	public static readonly Height = 1.32;
+	public actor: MRE.Actor = null;
+
+	public CreateActor(assetContainer: MRE.AssetContainer, baseUrl: string): MRE.Actor {
+		if (this.actor !== null)
```

Comment:
This might just be my lack of experience with TS, but why not just make this a constructor rather than needing to enforce it's not called twice?

Reply:
It's just symmetry from the other Card and Deck classes. Probably makes more sense to have it in the constructor, I just wanted to leave it simple. And then making sure it's not called twice is to enforce correctness down the line.